src/Data/Array/Indexed.idr

```diff
@@ -300,6 +300,15 @@ curLTE s lte = transitive lte $ ixLTE s
 0 curLT : (s : Ix (S m) n) -> LTE c (ixToNat s) -> LT c n
 curLT s lte = let LTESucc p := ixLT s in LTESucc $ transitive lte p
 
+||| We can drop n elements from an array. O(n)
+|||
+||| Note: If you only need a small portion of a potentially large
+|||       array the rest of which you no longer need, consider to
+|||       release the large array from memory by invoking `force`.
```

Comment:
This comment does not apply to `drop` as we create a new array by copying values.

Reply:
Great catch, this should be resolved via 2e6e15eb8606ab1b603fa2504cb54897305b2668.